manim/animation/animation.py

```diff
@@ -132,7 +132,7 @@ def interpolate_mobject(self, alpha):
 
     def interpolate_submobject(self, submobject, starting_sumobject, alpha):
         # Typically implemented by subclass
-        pass
+        raise NotImplementedError
```

Comment:
I believe this has to be left as `pass`. The codebase is counting on this not doing anything in some cases.

Reply:
yes, I was originally trying to find where that gets used :D. 

Sorry this is probably a really dumb question but how can I run your test suite locally?

I'm using XUbuntu and previously I was running one of the examples for testing (I know I'm stupid for not reading your testing Wiki before submitting this PR) but when I read that and ran `pytest` in the manim/ directory, I don't get the same error as is returned by your Ubuntu tests, I get invalid syntax on this import `from rich import print as printf` is this me being dumb?

Reply:
No, you are not stupid nor dumb :sweat_smile: 

The correct way of running the tests locally is by executing either `pytest` or `poetry run pytest` from the `manim/` directory, depending on how you installed manim. See [here](https://docs.manim.community/en/v0.1.0/contributing.html#polishing-your-changes-and-submitting-them-for-review).

If the problem persists, could you please show me the traceback or error you get?

Reply:
Okay it was my fault :facepalm: when I run `pytest` it defaults to the python2 version so I have to run `python3 -m pytest` instead which correctly fails the tests :upside_down_face:

Reply:
So I've reverted the change as you suggested. There is an alternative to pass which you may or may not prefer (which doesn't break everything) and that is 

`return NotImplemented` 

[Docs entry](https://docs.python.org/3/library/constants.html#NotImplemented)

but I didn't add that because it's normally used specifically for dunder methods and I felt like I was just changing something for the sake of changing it which isn't really a good thing. If you think it looks better/ is clearer than the original `pass` statement then it may be worth using.

Reply:
At this point, I would rather leave the `pass`. I feel like we will change that eventually, but maybe as part of a major rewrite at some point in the future.